shared/route-tree/render-route.js

```diff
@@ -113,12 +113,13 @@ function _RenderRoute ({routeDef, routeState, setRouteState, path}: _RenderRoute
 
     stack = childStack
     if (routeDef.containerComponent) {
+      const useKeys = stack.count() > 1 // Only use keys if we have siblings, else we'll render thrash if we keep unique state in here (like in chat)
       // If this route specifies a container component, compose it around every
       // view in the stack.
       stack = stack.map(r => (
         r.update('component', child => (
           <RenderRouteNode
-            key={pathToString(path)}
+            key={useKeys ? pathToString(path) : undefined}
```

Comment:
You can just delete the key here. I don't believe stack items are ever rendered as siblings.